packages/phoenix-otel/README.md

```diff
@@ -61,6 +61,15 @@ from phoenix.otel import register
 tracer_provider = register(endpoint="http://localhost:6006/v1/traces")
 ```
 
+Additionally, the `protocol` argument can be used to enforce the OTLP transport protocol
+regardless of the endpoint specified. This might be useful in cases such as when the GRPC
+endpoint is bound to a different port than the default (4317).
```

Comment:
provide the enum values of the protocol in the readme? Also reminder to update gitbook